docs/dev/install.rst

```diff
@@ -4,79 +4,114 @@ Development Installation
+   Take into account that this setup is intended for development purposes. We do not recommend to follow this guide to deploy an instance of Read the Docs for production.
+
+
+Installing Docker
+-----------------
+
+You need `Docker <https://www.docker.com/>`_ to run a development setup of Read the Docs. The easiest place to start is `Docker's installation guide <https://docs.docker.com/install/>`_, however there might be other more specific ways to get Docker for your exact system.
+
+If you run Windows, you can run Docker on `Windows Subsystem for Linux (WSL) <https://docs.microsoft.com/en-us/windows/wsl/tutorials/wsl-containers>`_ or `Docker Desktop for Windows <https://docs.docker.com/desktop/windows/install/>`_.
+
+   .. tip::
+
+      We always support Docker CE for the latest Ubuntu LTS. On Ubuntu 22.04, we are therefore supporting Docker 20.x, the version of the ``docker.io`` package.
+
+      The following will install Docker Community Edition maintained by a Ubuntu/Canonical team:
+
+      .. prompt:: bash
+
+         # Install the docker.io package
+         sudo apt install docker.io
+         # Add the current user to the Docker group. After this, you should log out and back in for the group membership to be effective.
+         sudo adduser $(whoami) docker
 
 
 Set up your environment
 -----------------------
 
-#. install `Docker <https://www.docker.com/>`_ following `their installation guide <https://docs.docker.com/install/>`_.
-
-#. clone the ``readthedocs.org`` repository:
+#. Clone the ``readthedocs.org`` repository:
 
    .. prompt:: bash
 
       git clone --recurse-submodules https://github.com/readthedocs/readthedocs.org/
 
-#. install the requirements from ``common`` submodule:
+#. Install the requirements from ``common`` submodule:
 
    .. prompt:: bash
 
       pip install -r common/dockerfiles/requirements.txt
 
-#. build the Docker image for the servers:
+#. Build the Docker image for the servers:
 
    .. warning::
 
       This command could take a while to finish since it will download several Docker images.
 
    .. prompt:: bash
 
-      inv docker.build
+      COMPOSE_PROJECT_NAME=community inv docker.build
 
    .. tip::
 
       If you pass the ``GITHUB_TOKEN`` and ``GITHUB_USER`` environment variables to this command,
       it will add support for readthedocs-ext.
 
-#. pull down Docker images for the builders:
+#. Pull down Docker images for the builders:
 
    .. prompt:: bash
 
       inv docker.pull --only-required
 
-#. start all the containers:
+#. Start all the containers:
 
    .. prompt:: bash
 
-      inv docker.up  --init  # --init is only needed the first time
+      COMPOSE_PROJECT_NAME=community inv docker.up  --init  # --init is only needed the first time
+
+#. Go to http://community.dev.readthedocs.io to access your local instance of Read the Docs. On newer versions of Firefox and Chrome, use a fresh Private/Incognito session.
```

Comment:
I think this happens if you have firefox HTTPS-Only mode enabled (I use a private window on FF bc of that), on chromium it works in a normal window for me (I just use chromium for testing)

Reply:
I faced this issue when I was setting up local environment. I think this happens because of browser HSTS settings (I think Read the Docs uses HSTS on production and it gets set on `*.readthedocs.io` domain). It works after you clear HSTS cache from the browser.

Reply:
Yeah. I've been there as well. I'd like to avoid mentioning incognito at this point and put a note about using `http://` instead. Then in the troubleshooting section, we can add an entry that explains how to clear the cache browser in case this problem is hit.

Reply:
I've elaborated the issue now. Let's get to #9310 after this documentation update so it doesn't get stuck? :)